use-timescale/schema-management/foreign-data-wrappers.md

```diff
@@ -8,16 +8,16 @@ tags: [change]
 
 # Foreign data wrappers
 
-You use foreign data wrappers (FDWs) to query external data sources from a Timescale Cloud service. These data sources can be one of the following:
+You use foreign data wrappers (FDWs) to query external data sources from a $SERVICE_LONG. These external data sources can be one of the following:
 
-- Other Timescale Cloud services.
-- PostgreSQL databases outside of Timescale Cloud.
+- $SERVICE_LONGs
+- PostgreSQL databases outside of $CLOUD_LONG
 
-If you are using [VPC peering][vpc-peering], you can create FDWs in your Customer VPC for a service in Timescale VPC or the same project. However, you can't create FDWs from Timescale VPC to Customer VPC.
+If you are using [$VPC peering][vpc-peering], you can configure FDWs in your Customer VPC to query a $SERVICE_SHORT in Timescale $VPC or the same project. However, you can't create FDWs in Timescale $VPC to query a data source in your Customer VPC.
```

Comment:
I would add something along the lines of:
This is because Timescale VPC peering uses AWS PrivateLink for increased security. Read the [VPC peering](https://docs.timescale.com/use-timescale/latest/security/vpc/) documentation for additional details.